awx/network_ui/static/network_ui/CONTRIBUTING.md

```diff
@@ -115,16 +122,11 @@ function Link(id, from_device, to_device, from_interface, to_interface) {
     this.name = "";
 }
 ```
+ 
+The following example sets the toolbox.selected_item value to the variable 
+item which the directives used in the child scope expect to be set.
 
-In some cases we need to set a variable in a child scope to a value.  In the
-scheme language this is called `let`. AngularJS is missing a ng-let directive
-but this can be simulated with ng-repeat and a list. The following example
-sets the toolbox.selected_item value to the variable item which the directives
-used in the child scope expect to be set.
-
-See: <https://docs.racket-lang.org/reference/let.html> (Optional reference, just look if you are curious)
-
-See: widgets/inventory_toolbox.html
```

Comment:
I removed this paragraph b/c the explanation of `let` detracts from your explanation of how you use ng-repeat to set variables on a child scope. Just my opinion though. JS has a `let` keyword so mixing that with ng-let (which doesn't exist) is kind of confusing in a front end context.